### PR TITLE
Rename Admin to Status

### PR DESCRIPTION
**Pull request by pmprete**
Add Exchange Rate and Reserves
Remove unsused functions and constants
Refactor try to use BigNumber div matissa as default
Remove APY column from status use APR instead
Show APY in dialong instead APR
Fix APR in supply dialogs
Fix Supply-Borrow Ratio in Status Dialog as it should been multiplied by 100
Fix Total Suplyed in Status Dialog as it was being shown in cToken and not in underlying
Filters now don't cast bigNumber to Number

### CODE REVIEW

File: src/components/dialog/repay/RepayInput.vue [hunk 1]
```diff
@@ -199,35 +199,27 @@ export default {
...
-        this.needApproval = !allow
-        return this.data.market.getMaxBorrowAllowed(this.account)
-      })
-      .then((maxBorrowAllowed) => {
-        this.maxBorrowAllowed = maxBorrowAllowed
-        return this.$middleware.getAccountHealth(this.account)
-      })
-      // sets health
-      .then((health) => {
-        this.accountHealth = health
```
Comment (fgregoribattista) [inline src/components/dialog/repay/RepayInput.vue:215]: why deleted accountHealt, not used in dialog(?)

File: src/components/status/MarketItem.vue [hunk 2]
```diff
@@ -135,38 +134,29 @@ export default {
...
+    this.totalBorrows = this.market.totalBorrows
+    this.loanToValue = this.market.loanToValue
   },
   methods: {
     reset() {
       this.dialog = false
       this.market
         .getPriceInDecimals()
-        .then((marketPrice) => {
+        .then(async (marketPrice) => {
```
Comment (fgregoribattista) [inline src/components/status/MarketItem.vue:147]: i dont understand this async funcion

**Review by fgregoribattista — APPROVED** [review]
just that comment and it's fine for me

File: src/components/status/MarketItem.vue (hunk 2 above)
Comment (pmprete) [reply src/components/status/MarketItem.vue:147]: this was from a previous intent, will remove it

**Pull request merged**